modules/tracking/include/opencv2/tracking/tracker.hpp

```diff
@@ -1227,12 +1227,12 @@ class CV_EXPORTS_W TrackerKCF : public Tracker
     Params();
 
     /**
-    * \brief Read parameters from file, currently unused
+    * \brief Read parameters from file
     */
     void read(const FileNode& /*fn*/);
 
     /**
-    * \brief Read parameters from file, currently unused
+    * \brief Write parameters in a file
```

Comment:
IMHO here should be `Write parameters to a file`

Reply:
IMHO, the principles to fix these wrong comments is as few changes as possible, at the same time, consistent with existed comments. In the line 1116 of tracker.hpp, there is a comment, i.e. `* \brief Write parameters in a file`, so I copyed it. If only from the view of its meaning, 'Write parameters to a file' should be better.